ios/fastlane/Fastfile

```diff
@@ -134,15 +135,57 @@ platform :ios do
         readonly: false,
         keychain_name: "tech_at_safepaths_keychain",
         keychain_password: "meow",
-        git_basic_authorization: ENV["secrets.MATCH_GIT_BASIC_AUTH"]
+        git_basic_authorization: ENV["secrets.MATCH_GIT_BASIC_AUTH"],
+        app_identifier: 'org.pathcheck.covid-safepaths'
+      )
+
+      build_ios_app(
+        workspace: WORKSPACE,
+        scheme: GPS_SCHEME,
+        configuration: "Release",
+        silent: true,
+        clean: true,
+        output_name: "COVIDSafePaths.ipa",
+      )
+  end
+
+    desc "Push a new BTE Release build to TestFlight"
+    lane :release_bte do
+      # Sets the app store version:
+      # https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion
+      increment_build_number(
+        xcodeproj: PROJECT,
+        build_number: JS_PACKAGE['version']
+      )
+      # Sets the system build number for the project:
+      # https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleshortversionstring
+      increment_version_number(
+        xcodeproj: PROJECT,
+        version_number: JS_PACKAGE['version']
+      )
+
+      signing_bundle_details = {
+        :BUNDLE_IDENTIFIER => BTE_APP_ID,
+        :PROVISIONING_PROFILE_SPECIFIER => BTE_PROFILE,
+        :DEVELOPMENT_TEAM => TEAM
+      }
+
+      match(
+        type: 'appstore',
+        readonly: false,
+        keychain_name: "tech_at_safepaths_keychain",
+        keychain_password: "meow",
+        git_basic_authorization: ENV["secrets.MATCH_GIT_BASIC_AUTH"],
+        app_identifier: 'org.pathcheck.bt'
```

Comment:
@mattThousand , @efalkner , I think this will solve the bundle id issue we bumped up into for the various flavors/targets.  Curious on you thoughts here.